### PR TITLE
[author] [new] add ami.js v0.0.11 w/ npm auto-update

### LINKED ISSUE #9763: [Request] Add ami.js
**Library name:** ami.js
**Git repository url:** https://github.com/FNNDSC/ami
**npm package url(optional):** https://www.npmjs.com/package/ami.js 
**License(s):** Apache2
**Official homepage:** https://github.com/FNNDSC/ami

### PR DESCRIPTION
**Pull request by NicolasRannou**
Pull request for issue: #9763 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/FNNDSC/ami
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/ami.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 45
   - Count of watchers: 16
   - Count of forks: 17
 * NPM download stats (optional):
   - Downloads in the last day:0
   - Downloads in the last week:24
   - Downloads in the last month:265

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

Closes #9763

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@NicolasRannou would you please add some keywords in `package.json` and update the commit message to something like the updated PR title? Thanks.

**Comment by kennynaoh** [comment]
Hi @NicolasRannou would you like to add https://fnndsc.github.io/ami/ as your `homepage` field in `package.json`?  Thanks!

**Comment by NicolasRannou** [comment]
@PeterDaveHello @kennynaoh all set :)

**Comment by PeterDaveHello** [comment]
@NicolasRannou Thanks!

**Comment by PeterDaveHello** [comment]
@NicolasRannou would you please also rebase this branch with the latest master branch? Thanks!

**Comment by PeterDaveHello** [comment]
Please also add `close #9763` in the commit message, thanks.

**Comment by NicolasRannou** [comment]
OK hopefully we are all good now - it is a bit of a pain to work with git as every single command on my machine takes soooo much time in the cdnjs repo. :)

**Comment by PeterDaveHello** [comment]
@NicolasRannou Yeah hope we could merge it this time, just asking, have you enabled git sparsecheckout?

**Comment by NicolasRannou** [comment]
Yes I think I did:
```
nico@vpn-172-18-192-147:~/work/gitroot/cdnjs (ami.js)$ ls
ajax
nico@vpn-172-18-192-147:~/work/gitroot/cdnjs (ami.js)$ 
```

**Comment by NicolasRannou** [comment]
@PeterDaveHello looks like CI tests are failing but not sure if that is relevant

File: ajax/libs/ami.js/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  ],
+  "homepage": "https://fnndsc.github.io/ami",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FNNDSC/ami"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name" : "Nicolas Rannou",
+    "url" : "https://github.com/NicolasRannou"
```
Comment (PeterDaveHello) [inline ajax/libs/ami.js/package.json]: oops, wrong indent here, no space before `:` please :) should pass the test this time

**Comment by PeterDaveHello** [comment]
@NicolasRannou There is a bug inside the drone ci caused the error message not outputed properly, sorry about that'

**Comment by NicolasRannou** [comment]
Thanks @PeterDaveHello @kennynaoh

**Comment by PeterDaveHello** [comment]
@NicolasRannou You're welcome.

**Pull request merged**